recipes/nbval/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "nbval" %}
+{% set version = "0.5" %}
+{% set sha256 = "bd245c0fb3ade4aff63948d31de5cb5f92129c7387e597d1229f687e5350eba7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Yeah, not yet. We need to migrate to `conda` 4.3 and make some changes to `conda-smithy`, but it should be doable.